src/ontology/mondo-edit.obo

```diff
@@ -430962,6 +431118,54 @@ subset: ordo_group_of_disorders
 xref: Orphanet:596426 {source="MONDO:equivalentTo"}
 is_a: MONDO:0002254 {source="https://orcid.org/0000-0001-5208-3432", source="https://orcid.org/0000-0002-4142-7153"} ! syndromic disease
 
+[Term]
+id: MONDO:0035694
+name: combined immunodeficiency due to RELA haploinsufficiency
+subset: ordo_disorder
+synonym: "CID due to RELA haploinsufficiency" EXACT [Orphanet:596759]
+xref: Orphanet:596759 {source="MONDO:equivalentTo"}
+is_a: MONDO:0015131 {source="https://orcid.org/0000-0001-5208-3432"} ! combined immunodeficiency
+
+[Term]
+id: MONDO:0035696
+name: incomplete septal cirrhosis
+def: "A histopathological form of portosinusoidal vascular disease characterized by the presence of incomplete, thin, perforated, or blind-ended septa, which intermittently delimit rudimentary nodules, although complete cirrhotic-type regenerative nodules are not seen. Isolated collagen bundles can also be observed within the parenchyma." [Orphanet:596941]
+subset: ordo_subtype_of_a_disorder
+synonym: "Incomplete septal fibrosis" EXACT [Orphanet:596941]
+xref: ICD10:K74.6 {source="Orphanet:596941", source="MONDO:MONDO:mondoIsNarrowerThanSource"}
+xref: Orphanet:596941 {source="MONDO:equivalentTo"}
+is_a: MONDO:0035357 {source="https://orcid.org/0000-0001-5208-3432"} ! portosinusoidal vascular disease
+relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql/qc/general/qc-single-child.sparql
+
+[Term]
+id: MONDO:0035706
+name: SETD2-related microcephaly-severe intellectual disability-multiple congenital anomalies syndrome
+subset: ordo_disorder
+xref: Orphanet:597743 {source="MONDO:equivalentTo"}
+is_a: MONDO:0002254 {source="https://orcid.org/0000-0001-5208-3432"} ! syndromic disease
+
+[Term]
+id: MONDO:0035707
+name: blepharophimosis-intellectual disability syndrome/genitopatellar overlap syndrome
+subset: ordo_disorder
+xref: Orphanet:597746 {source="MONDO:equivalentTo"}
+is_a: MONDO:0002254 {source="https://orcid.org/0000-0001-5208-3432"} ! syndromic disease
+
+[Term]
+id: MONDO:0035713
+name: FOXG1 syndrome due to intragenic alteration
+subset: ordo_subtype_of_a_disorder
+xref: Orphanet:598164 {source="MONDO:equivalentTo"}
+is_a: MONDO:0100040 {source="https://orcid.org/0000-0001-5208-3432"} ! FOXG1 disorder
```

Comment:
I am not sure this is correct... but maybe it is ok: 
Is FOXG1 syndrome a FOXG1 disorder? 
Since "syndrome" is a child of "disorder", I think it is probably ok since 'disease or disorder' is the very high level term of the whole ontology)? 
(I did not add my orcid because I want to know that 'syndrome' is indeed a 'disorder')

Reply:
these comments have all shifted, but I believe this is in response to 
id: MONDO:0035713
name: FOXG1 syndrome due to intragenic alteration

I'll leave it for now.